features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/NewOverview/NewCustomizedStepper.jsx

```diff
@@ -175,6 +178,125 @@ export default function NewCustomizedStepper() {
             });
     }
 
+    /**
+     * This function renders the final lifecycle state
+     * @param {*} state
+     */
+    function finalLifecycleState(state) {
+        switch (state) {
+            case 'PUBLISHED':
+                return (
+                    <>
+                        <Grid
+                            container
+                            direction='row'
+                            alignItems='center'
+                            justify='center'
+                        >
+                            <Grid item>
+                                <CheckIcon className={classes.iconTrue} />
+                            </Grid>
+                            <Box ml={1}>
+                                <Grid item>
+                                    <Typography variant='h7'>
+                                        <FormattedMessage
+                                            id='Apis.Details.Overview.CustomizedStepper.publish'
+                                            defaultMessage=' Published (Current API)'
```

Comment:
This current API should have been highlighted and should have a view in dev portal link

![image](https://user-images.githubusercontent.com/3313885/112423878-733e4a80-8d59-11eb-8b56-2f7ffbd3c0df.png)

refer : https://xd.adobe.com/view/be3fc799-0d72-473d-a515-9f7665b5d08a-eb6c/screen/0ca95095-508e-4616-961b-08c38998d6b0

Reply:
Fixed